packages/gitlab-backend/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './service/router';
 export * from './processor';
+export { gitlabPlugin as default } from './plugin';
```

Comment:
Here is missing the `catalogPluginGitlabFillerProcessorModule`, I think it is better to export it as:
```ts
export * from './plugin';
```

Reply:
@antoniomuso Updated.